reference/safe-core-sdk/onramp-kit/monerium.md

```diff
@@ -0,0 +1,151 @@
+## Monerium
+
+The Monerium pack enables you to take advantage of Safe with Monerium services. [Monerium](https://monerium.com) is a regulated platform that facilitates the use of e-money tokens on the blockchain.
+
+### Install
+
+If you choose to use the `MoneriumPack`, you must install additional dependencies in addition to the `@safe-global/onramp-kit` package.
+
+Install the `@monerium/sdk` package:
+
+```bash
+yarn add @monerium/sdk @safe-global/onramp-kit
+```
+
+### MoneriumPack
+
+The `MoneriumPack` class enables the use of Monerium services with Safe. To use it, create an instance of the pack and pass it to the `SafeOnRampKit` `init` method.
+
+This pack makes possible to "Login with Monerium" by creating a connection between your Safe address and your Monerium account. Using this pack you will start an authentication flow using the Monerium SDK which will allow you to gain access to your account resources.
+
+#### `new MoneriumPack(moneriumConfig)`
+
+```typescript
+const moneriumPack = new MoneriumPack(moneriumConfig);
+await SafeAuthKit.init(moneriumPack);
+```
+
+**Params**
+
+- `moneriumConfig` - The configuration for the Monerium pack. The options are:
+
+```typescript
+MoneriumConfig {
+  clientId: string
+  environment: 'production' | 'sandbox'
+}
+```
+
+The `clientId` is the secret representing the "Authorization Code Flow" for your Monerium account. You can obtain one by logging into [your account](https://monerium.dev) and creating a [new application](https://monerium.dev/docs/getting-started/create-app).
```

Comment:
Is the `clientId` actually a secret? Since it's being used client side it's not technically a secret.

```suggestion
The `clientId` is the secret representing the "Authorization Code Flow" for your Monerium account. To get your `clientId`:

1. Log in to [your account](https://monerium.dev) 
2. Create a [new application](https://monerium.dev/docs/getting-started/create-app).
```